packs/actions/overwhelming-combination.json

```diff
@@ -11,14 +11,22 @@
         },
         "category": "offensive",
         "description": {
-            "value": "<p><strong>Requirements</strong> You're wielding a one-handed melee weapon or a melee weapon with the agile or finesse trait</p><hr /><p><strong>Effect</strong> s Make two Strikes against a target within your reach, one with the required weapon and one with your fist unarmed attack. If both hit the same target, combine their damage for the purposes of its resistances and weaknesses. Apply your multiple attack penalty to each Strike normally.</p>"
+            "value": "<p><strong>Requirements</strong> You're wielding a one-handed melee weapon or a melee weapon with the agile or finesse trait</p><hr /><p><strong>Effects</strong> Make two Strikes against a target within your reach, one with the required weapon and one with your fist unarmed attack. If both hit the same target, combine their damage for the purposes of its resistances and weaknesses. Apply your multiple attack penalty to each Strike normally.</p>"
         },
         "publication": {
             "license": "ORC",
             "remaster": true,
             "title": "Pathfinder Lost Omens: Tian Xia Character Guide"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "RollOption",
+                "option": "overwhelming-combination",
+                "predicate": [
+                    "feat:kaiju-defense-oath"
+                ]
+            }
```

Comment:
I think this is missing a toggle, it doesn't look like it's supposed to be on all strikes since it has the flourish trait you can't use it more than once per round.

Reply:
It is indeed missing a toggle